services/color-formatters.js

```diff
@@ -175,24 +175,26 @@ function colorScale(steps, colors, reversed) {
 /**
  * Determines the color used for a badge according to the age.
  * Age is calculated as days elapsed till current date.
- * The color varies from bright green to red as the age increases.
+ * The color varies from bright green to red as the age increases
+ * or the other way around if `reverse` is given `true`.
  *
  * @param {string} date Date string
+ * @param {boolean} reversed Reverse the color scale a.k.a. the older, the better
  * @returns {string} Badge color
  */
-function age(date) {
-  const colorByAge = colorScale([7, 30, 180, 365, 730], undefined, true)
+function age(date, reversed = false) {
+  const colorByAge = colorScale([7, 30, 180, 365, 730], undefined, !reversed)
   const daysElapsed = dayjs().diff(dayjs(date), 'days')
   return colorByAge(daysElapsed)
 }
```

Comment:
Added a `reversed` parameter with default value to `false`, which is passed to `colorScale` function by reversing in again with `!reversed`. I've thought re-reversing this boolean value is appropriate, semantically-speaking.